packages/@react-native-windows/cli/src/runWindows/runWindowsOptions.ts

```diff
@@ -102,6 +102,7 @@ export const runWindowsOptions: CommandOption[] = [
   {
     name: '--no-packager',
     description: 'Do not launch packager while building',
+    default: false,
```

Comment:
When defining a boolean parameter with the `--no-` convention, commander completely ignores anything you set as default.

In my telemetry PR #8253, I add unit tests to make sure we don't set defaults because it's misleading.

Reply:
That's annoying... Will remove this line.